packages/bierzo-wallet/src/components/Header/index.stories.tsx

```diff
@@ -57,8 +57,11 @@ const txs: readonly (ProcessedSendTransaction | ProcessedTx)[] = [
 ];
 
 const txStore: DeepPartial<RootState> = {
+  // REVIEW Type '{ transactions: readonly (ProcessedSendTransaction | ProcessedTx<LightTransaction>)[]; }'
+  // is not assignable to type 'DeepPartial<NotificationState>'.
+  // Type 'TransactionId' is not assignable to type 'undefined'
   notifications: {
-    transactions: txs,
+    transactions: txs as any,
```

Comment:
Should I leave this as is? @webmaster128 @albertandrejev

Reply:
I don't mind. Next time we work with this code, we can look into.